### ISSUE TITLE
build dependencies did not run successfully because there is no compatible version of numpy

### ISSUE THREAD

**Issue by 7w01** [salesforce/LAVIS#754]
> ERROR: Ignored the following versions that require a different python version: 1.25.0 Requires-Python >=3.9; 1.25.1 Requires-Python >=3.9; 1.25.2 Requires-Python >=3.9; 1.26.0 Requires-Python <3.13,>=3.9; 1.26.1 Requires-Python <3.13,>=3.9; 1.26.2 Requires-Python >=3.9; 1.26.3 Requires-Python >=3.9; 1.26.4 Requires-Python >=3.9; 2.0.0 Requires-Python >=3.9; 2.0.1 Requires-Python >=3.9; 2.0.2 Requires-Python >=3.9; 2.1.0 Requires-Python >=3.10; 2.1.0rc1 Requires-Python >=3.10; 2.1.1 Requires-Python >=3.10; 2.1.2 Requires-Python >=3.10
                  ERROR: Could not find a version that satisfies the requirement numpy<3.0.0,>=2.0.0 (from versions: 1.3.0, 1.4.1, 1.5.0, 1.5.1, 1.6.0, 1.6.1, 1.6.2, 1.7.0, 1.7.1, 1.7.2, 1.8.0, 1.8.1, 1.8.2, 1.9.0, 1.9.1, 1.9.2, 1.9.3, 1.10.0.post2, 1.10.1, 1.10.2, 1.10.4, 1.11.0, 1.11.1, 1.11.2, 1.11.3, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 1.13.3, 1.14.0, 1.14.1, 1.14.2, 1.14.3, 1.14.4, 1.14.5, 1.14.6, 1.15.0, 1.15.1, 1.15.2, 1.15.3, 1.15.4, 1.16.0, 1.16.1, 1.16.2, 1.16.3, 1.16.4, 1.16.5, 1.16.6, 1.17.0, 1.17.1, 1.17.2, 1.17.3, 1.17.4, 1.17.5, 1.18.0, 1.18.1, 1.18.2, 1.18.3, 1.18.4, 1.18.5, 1.19.0, 1.19.1, 1.19.2, 1.19.3, 1.19.4, 1.19.5, 1.20.0, 1.20.1, 1.20.2, 1.20.3, 1.21.0, 1.21.1, 1.21.2, 1.21.3, 1.21.4, 1.21.5, 1.21.6, 1.22.0, 1.22.1, 1.22.2, 1.22.3, 1.22.4, 1.23.0, 1.23.1, 1.23.2, 1.23.3, 1.23.4, 1.23.5, 1.24.0, 1.24.1, 1.24.2, 1.24.3, 1.24.4)
                  ERROR: No matching distribution found for numpy<3.0.0,>=2.0.0

I tried different version of python3.8, none of them worked

**Comment by hhhheep** [comment]
may be you can try this https://github.com/salesforce/LAVIS/issues/641

**Comment by AnnieQian1212** [comment]
Same error

**Comment by huizyuan** [comment]
same issue

**Comment by WindWhisperOvO** [comment]
I encountered the same issue. It seems that the package salesforce-lavis is supported on python>=3.9 now. If your pip is bounded with python 3.8, numpy version can not be satisfied. 

So try this:
```bash
pip install numpy==2.0.1
```
If it fails, you should use the higher version of python. And then install salesforce-lavis:
```bash
pip install salesforce-lavis
```
I think it is helpful to you.

**Comment by yearnallover** [comment]
> I encountered the same issue. It seems that the package salesforce-lavis is supported on python>=3.9 now. If your pip is bounded with python 3.8, numpy version can not be satisfied.
> 
> So try this:
> 
> pip install numpy==2.0.1
> If it fails, you should use the higher version of python. And then install salesforce-lavis:
> 
> pip install salesforce-lavis
> I think it is helpful to you.

It means just using the higher python version can be all right?

**Comment by yearnallover** [comment]
> > I encountered the same issue. It seems that the package salesforce-lavis is supported on python>=3.9 now. If your pip is bounded with python 3.8, numpy version can not be satisfied.
> > So try this:
> > pip install numpy==2.0.1
> > If it fails, you should use the higher version of python. And then install salesforce-lavis:
> > pip install salesforce-lavis
> > I think it is helpful to you.
> 
> It means just using the higher python version can be all right?

Yes, I just change to 3.9, no error occurs anymore while installing.
